include/z64audio.h

```diff
@@ -526,10 +139,10 @@ typedef struct {
     /* 0x0004 */ u16 unk_4;
     /* 0x0006 */ char unk_0006[0xA];
     /* 0x0010 */ s16* adpcmCodeBook;
-    /* 0x0014 */ NoteSampleState* sampleStateList;
+    /* 0x0014 */ struct NoteSampleState* sampleStateList;
     /* 0x0018 */ SynthesisReverb synthesisReverbs[4];
     /* 0x0B58 */ char unk_0B58[0x30];
-    /* 0x0B88 */ Sample* usedSamples[128];
+    /* 0x0B88 */ struct Sample* usedSamples[128];
```

Comment:
Oh... we have a 128 max sample workspace? and that includes bank 00/01/02? and the game might have two banks one for bgm and one for combat? hmm..

Reply:
There are actually many in-built ways to store samples, and pretty sure this `usedSamples` is an entirely unused system in both MM and OoT. The name was just copied over from OoT so I don't recall exactly how this system was designed, just that it's unused

Reply:
oh it's unused? interesting